src/main/java/ru/my/Mocito/departmentController/DepartmentController.java

```diff
@@ -0,0 +1,48 @@
+package ru.my.Mocito.departmentController;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ru.my.Mocito.Employee;
+import ru.my.Mocito.departmentService.DepartmentImpl;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/department")
+public class DepartmentController {
+    private final DepartmentImpl departmentimpl;
+
+    public DepartmentController(DepartmentImpl departmentimpl) {
```

Comment:
```suggestion
    public DepartmentController(Idepartment departmentimpl) {
```
инъекцию надо делать по интерфейсу, а не реализации